source/base/Text.ooc

```diff
@@ -182,6 +182,16 @@ Text: cover {
 		printf("%s\n", this _buffer _backend pointer as Char*)
 		this free(Owner Receiver)
 	}
+	format: func (args: ...) -> This {
+		t := this take()
+		string := t toString()
```

Comment:
It depends on `String` for now, but at least `DateTime` and `TimeSpan` can be implemented without `String` now.